pinot-core/src/main/java/org/apache/pinot/core/geospatial/transform/function/ScalarFunctions.java

```diff
@@ -43,6 +44,23 @@
         .serialize(GeometryUtils.GEOMETRY_FACTORY.createPoint(new Coordinate(x, y)));
   }
 
+  /**
+   * Creates a point.
+   *
+   * @param x x
+   * @param y y
+   * @param isGeography if it's geography
+   * @return the created point
+   */
+  @ScalarFunction
+  public static byte[] stPoint(double x, double y, double isGeography) {
```

Comment:
Does `boolean` work as argument? If not, `int` might be a better option?

Reply:
Unfortunately it does not.... The PQL parser will invoke this scalar function and pass in the constant. Since boolean is not available in PQL/SQL, we need to use a number. Also, somehow the parser parses `1` into a double type, any idea on that?

Reply:
The parser parses the value based on the method signature. If you change the third argument to `int`, it should be able to parse `1` to `int`